.github/workflows/main.yml

```diff
@@ -27,86 +18,44 @@ jobs:
       - name: "Install PHP"
         uses: "shivammathur/setup-php@2.4.3"
         with:
-          coverage: "none"
-          php-version: "${{ matrix.php-version }}"
-
-      - name: "Validate composer.json and composer.lock"
-        run: "composer validate --strict"
+          coverage: none
+          php-version: "7.4"
+          tools: composer:v2
 
-      - name: "Determine composer cache directory"
-        id: "determine-composer-cache-directory"
-        run: "echo \"::set-output name=directory::$(composer config cache-dir)\""
-
-      - name: "Cache dependencies installed with composer"
-        uses: "actions/cache@v2.1.1"
-        with:
-          path: "${{ steps.determine-composer-cache-directory.outputs.directory }}"
-          key: "php-${{ matrix.php-version }}-composer-${{ matrix.dependencies }}-${{ hashFiles('composer.lock') }}"
-          restore-keys: "php-${{ matrix.php-version }}-composer-${{ matrix.dependencies }}-"
+      - name: Validate composer.json
+        run: composer validate --strict
 
-      - name: "Install lowest dependencies from composer.json"
-        if: "matrix.dependencies == 'lowest'"
-        run: "composer update --no-interaction --no-progress --no-suggest --prefer-lowest"
 
-      - name: "Install locked dependencies from composer.lock"
-        if: "matrix.dependencies == 'locked'"
-        run: "composer install --no-interaction --no-progress --no-suggest"
+      - name: Install dependencies
+        run: composer install --prefer-dist --no-interaction --no-progress --no-suggest
 
-      - name: "Install highest dependencies from composer.json"
-        if: "matrix.dependencies == 'highest'"
-        run: "composer update --no-interaction --no-progress --no-suggest"
 
-      - name: "Run friendsofphp/php-cs-fixer"
-        run: "vendor/bin/php-cs-fixer fix --config=.php_cs --diff --diff-format=udiff --dry-run --verbose"
+      - name: Run friendsofphp/php-cs-fixer
+        run: composer run format -- --config=.php_cs --diff --diff-format=udiff --dry-run --verbose
```

Comment:
I moved this to the composer scripts section that it isn't needed to call the 'vendor/bin/php-cs-fixer' path directly.